snyk-monitor/templates/deployment.yaml

```diff
@@ -18,23 +18,10 @@ spec:
         app.kubernetes.io/name: {{ include "snyk-monitor.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
+      securityContext:
+        fsGroup: 2000
```

Comment:
Unfortunately I don't think this works on OpenShift, the group IDs are randomly allocated per namespace and they're some really high numbers (in the millions). We should have a look into OS's documentation regarding these 🤔

Reply:
The group doesn't have to exist for this to work. Kubernetes actually creates the `fsGroup` when the pod starts and assigns it as a supplimentary group to all running processes. The groupIds that OS generates won't matter.